common/web/lm-worker/src/main/correction/context-tracker.ts

```diff
@@ -126,7 +151,7 @@ export class TrackedContextState {
   }
 
   popHead() {
-    this.tokens.splice(0, 2);
+    this.tokens.splice(0, 1);
```

Comment:
What's this change about?

Reply:
Originally, the context tracker assumed there was always (untracked) whitespace between tokens.  We modeled it by inserting an extra instance to represent the whitespace.

Now that we're actually _tracking_ the whitespace, we can just use the real tokenization instead.  Dropping the "extra instance insertion" is the reason for the decrement.